docs/getting-started/uninstall/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Uninstall Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/uninstall/"
+---
+
+Shopify App CLI can be removed from your system using the same package manager you used to install it.
+
+### Brew (MacOS)
+
+```console
+$ brew uninstall shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+```console
+$ sudo apt remove shopify-cli
+```
+
+### RPM (CentOS, Fedora, Red Hat, SUSE)
```

Comment:
To align with heading in install page - "RPM-based Linux distributions (e.g., Fedora, CentOS, SUSE)"